### ISSUE TITLE
downloads link should not show prereleases

### ISSUE THREAD

**Issue by eprenamer** [tvrenamer/tvrenamer.github.com#2]
Currently, http://tvrenamer.org/downloads redirects to https://github.com/tvrenamer/tvrenamer/releases

This takes the user to the page showing pre-releases.

I think it would be better to go to
https://github.com/tvrenamer/tvrenamer/releases/latest

which only shows the latest actual release.

I'm not sure that's ideal.  I think it would be nice to give them information about the various releases available -- maybe not redirect at all, but have a page on tvrenamer.org.  But as a quick solution, I definitely think `latest` is better than the main releases page.

**Comment by eprenamer** [comment]
@vipuldelwadia

**Comment by vipuldelwadia** [comment]
Just a note that the current downloads url http://tvrenamer.org/downloads isn't linked from anywhere, so I'm not sure how much traffic it actually gets. I imagine most people either go directly to github or download from the main page on http://tvrenamer.org/

**Comment by eprenamer** [comment]
No, it is linked from somewhere!  Assuming the user has "checkForUpdates" enabled in their prefs, when TVRenamer detects a new version is available, it enables a link on the toolbar to download the latest version.  That link goes to http://tvrenamer.org/downloads

I have verified everything is working.  I downloaded 0.7.2 and ran it, and it said there was an update available, and I clicked it, and it took me to the page showing the latest release, 0.8.
